backend/curriculum_tracking/api_views.py

```diff
@@ -838,12 +841,18 @@ class WorkshopAttendanceViewset(viewsets.ModelViewSet):
     ]
 
 
-class ManagmentActionsViewSet(viewsets.ViewSet):
+class ManagmentActionsViewSet(viewsets.ViewSet, GenericAPIView):
+
     serializer_class = serializers.NoArgs
+    queryset = models.Team.objects.filter(active=True)
 
     def list(self, request):
         return Response([])
 
+    def get_object(self):
+        queryset = self.get_queryset()
+        return queryset
```

Comment:
We can just return it;
```suggestion
    def get_object(self):
        return self.get_queryset()
```

Reply:
I changed this to your way.

Reply:
I hear you about the extra test but at the moment I can't do that unless that is what this new function should be doing.  The instructions say that we select the correct cards according to team, content_item and flavours.  Therefore, we are updating the due times of cards and not of projects.  The functionality is built to update cards and not projects.  If that should change then I can have a test that updates due_times for projects?